macOS: 2.93 fails to build with a lot of errors (2.92 built perfectly fine)

```
--->  Building asymptote
Executing:  cd "/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_graphics_asymptote/asymptote/work/asymptote-2.93" && /usr/bin/make -j6 -w all 
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_graphics_asymptote/asymptote/work/asymptote-2.93'
Creating win32helpers.d
Creating xstream.d
Creating win32xdr.d
Creating main.d
Creating win32pipestream.d
Creating symbolmaps.d
Creating lspserv.d
Creating GLTextures.d
Creating EXRFiles.d
Creating v3dfile.d
Creating jsfile.d
Creating shaders.d
Creating glrender.d
Creating tr.d
Creating predicates.d
Creating Delaunay.d
Creating memory.d
Creating array.d
Creating constructor.d
Creating asyprocess.d
Creating envcompleter.d
Creating refaccess.d
Creating fundec.d
Creating varinit.d
Creating application.d
Creating program.d
Creating asyparser.d
Creating locate.d
Creating impdatum.d
Creating coenv.d
Creating coder.d
Creating simpson.d
Creating parallel.d
Creating fftw++asy.d
Creating fileio.d
Creating interact.d
Creating record.d
Creating absyn.d
Creating virtualfieldaccess.d
Creating access.d
Creating camp.tab.d
Creating exithandlers.d
Creating stack.d
Creating newexp.d
Creating exp.d
Creating entry.d
Creating symbol.d
Creating name.d
Creating callable.d
Creating errormsg.d
Creating dec.d
Creating stm.d
Creating genv.d
Creating env.d
Creating tinyexr_impl.d
Creating writePRC.d
Creating PRCdouble.d
Creating oPRCFile.d
Creating PRCbitStream.d
Generating gsl.d from gsl.cc
Generating builtin.d from builtin.cc
Generating types.d from types.cc
perl ./opsymbols.pl --campfile camp.l --output opsymbols.h
Creating pipestream.d
Creating pen.d
Creating bezierpatch.d
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Creating beziercurve.d
Creating drawsurface.d
Creating drawpath3.d
Creating path3.d
Creating drawfill.d
Creating knot.d
Creating flatguide.d
Creating guide.d
Creating settings.d
Creating util.d
Creating texfile.d
Creating psfile.d
Creating picture.d
Creating drawlabel.d
Creating drawpath.d
Creating path.d
Creating camperror.d
Creating lex.yy.d
perl ./runtime.pl --prefix runmath --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runstring --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runpath3d --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runpath --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runtriple --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runpair --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runsystem --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runfile --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runarray --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runhistory --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runlabel --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runpicture --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runbacktrace --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
perl ./runtime.pl --prefix runtime --opsym-file opsymbols.h \
		--runtime-base-file runtimebase.in \
		--src-template-dir . \
		--header-out-dir . \
		--src-out-dir .
Generating runpath3d.d from runpath3d.cc
Generating runmath.d from runmath.cc
Generating runtriple.d from runtriple.cc
Generating runpath.d from runpath.cc
Generating runstring.d from runstring.cc
Generating runpair.d from runpair.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runsystem.d from runsystem.cc
Generating runfile.d from runfile.cc
Generating runarray.d from runarray.cc
Generating runhistory.d from runhistory.cc
Generating runlabel.d from runlabel.cc
Generating runpicture.d from runpicture.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runbacktrace.d from runbacktrace.cc
Generating runtime.d from runtime.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
make: Leaving directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_graphics_asymptote/asymptote/work/asymptote-2.93'
make: Entering directory `/opt/local/var/macports/build/_opt_PPCSnowLeopardPorts_graphics_asymptote/asymptote/work/asymptote-2.93'
Generating runtime.raw.i from runtime.cc
Generating runbacktrace.raw.i from runbacktrace.cc
Generating runpicture.raw.i from runpicture.cc
Creating exp.d
Generating runlabel.raw.i from runlabel.cc
Generating runhistory.raw.i from runhistory.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runarray.raw.i from runarray.cc
Generating runfile.raw.i from runfile.cc
Generating runsystem.raw.i from runsystem.cc
Generating runpair.raw.i from runpair.cc
Generating runtriple.raw.i from runtriple.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runpath.raw.i from runpath.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runpath3d.raw.i from runpath3d.cc
Generating runstring.raw.i from runstring.cc
Generating runmath.raw.i from runmath.cc
Generating types.raw.i from types.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating builtin.raw.i from builtin.cc
Generating gsl.raw.i from gsl.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Creating stm.d
Generating gsl.d from gsl.cc
Generating builtin.d from builtin.cc
Generating types.d from types.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runmath.d from runmath.cc
Generating runstring.d from runstring.cc
Generating runpath3d.d from runpath3d.cc
Generating runtriple.d from runtriple.cc
Generating runpath.d from runpath.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runpair.d from runpair.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runsystem.d from runsystem.cc
Generating runfile.d from runfile.cc
Generating runarray.d from runarray.cc
Generating runhistory.d from runhistory.cc
Generating runlabel.d from runlabel.cc
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runpicture.d from runpicture.cc
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
Generating runbacktrace.d from runbacktrace.cc
Generating runtime.d from runtime.cc
Creating camp.tab.d
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
gen_preprocessed_depfile.py: error: unrecognized arguments: /opt/local/bin/g++-mp-14
usage: gen_preprocessed_depfile.py [-h] --out-i-file OUT_I_FILE
                                   [--out-dep-file OUT_DEP_FILE] --in-src-file
                                   IN_SRC_FILE --cxx-compiler CXX_COMPILER
                                   [--msvc] [--include-dirs INCLUDE_DIRS]
                                   [--additional-raw-arguments ADDITIONAL_RAW_ARGUMENTS]
                                   [--macro-defs MACRO_DEFS]
                                   [--cxx-standard CXX_STANDARD]
                                   [--dep-file-only]
. . .
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o camperror.o -c camperror.cc
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o path.o -c path.cc
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o drawpath.o -c drawpath.cc
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o drawlabel.o -c drawlabel.cc
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o picture.o -c picture.cc
ccache /opt/local/bin/g++-mp-14 -DHAVE_CONFIG_H  -D_THREAD_SAFE -pthread -DFFTWPP_SINGLE_THREAD -Wall -isystem/opt/local/include/LegacySupport -I/opt/local/include -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR  -D_DARWIN_C_SOURCE -DNCURSES_WIDECHAR   -I/opt/local/include/eigen3  -I. -I/opt/local/include/gc -I/usr/include/gc -Iacextlibs/vectorgraphics/LspCpp/include -Iacextlibs/syoyo/tinyexr  -Ibackports/glew/include  -pipe -Os -std=c++14 -D_GLIBCXX_USE_CXX11_ABI=0 -arch ppc -pipe -Os -arch ppc -fno-var-tracking -Ibackports/optional/include -Iprc/include -o psfile.o -c psfile.cc
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from vm.h:11,
                 from camperror.cc:15:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from camperror.h:16,
                 from camperror.cc:14:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
make: *** [camperror.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from stack.h:13,
                 from mathop.h:14,
                 from path.cc:15:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from mod.h:13,
                 from path.h:16,
                 from path.cc:11:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
make: *** [path.o] Error 1
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from vm.h:11,
                 from array.h:11,
                 from pen.h:17,
                 from psfile.h:19,
                 from psfile.cc:14:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from pair.h:22,
                 from psfile.h:16:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
make: *** [psfile.o] Error 1
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from vm.h:11,
                 from array.h:11,
                 from pen.h:17,
                 from drawelement.h:16,
                 from drawpath.h:11,
                 from drawpath.cc:12:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from drawelement.h:13:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
In file included from drawelement.h:18:
texfile.h:60:1: error: invalid use of template-name 'std::unordered_set' without an argument list
   60 | std::unordered_set const latexCharacters = {'#', '$', '%', '&', '\\', '^', '_', '{', '}', '~'};
      | ^~~
texfile.h:60:1: note: class template argument deduction is only available with '-std=c++17' or '-std=gnu++17'
In file included from /opt/local/include/gcc14/c++/unordered_set:41,
                 from symbolmaps.h:4:
/opt/local/include/gcc14/c++/bits/unordered_set.h:104:11: note: 'template<class _Value, class _Hash, class _Pred, class _Alloc> class std::unordered_set' declared here
  104 |     class unordered_set
      |           ^~~~~~~~~~~~~
texfile.h: In function 'void camp::texpreamble(T&, mem::list<std::basic_string<char, std::char_traits<char>, gc_allocator_ignore_off_page<char> > >&, bool, bool)':
texfile.h:74:59: error: 'latexCharacters' was not declared in this scope; did you mean 'escapeCharacters'?
   74 |   out << "\\def\\ASYprefix{" << escapeCharacters(outPath, latexCharacters) << "}" << newl
      |                                                           ^~~~~~~~~~~~~~~
      |                                                           escapeCharacters
make: *** [drawpath.o] Error 1
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from vm.h:11,
                 from array.h:11,
                 from pen.h:17,
                 from drawelement.h:16,
                 from drawlabel.h:11,
                 from drawlabel.cc:12:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from drawelement.h:13:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
In file included from drawelement.h:18:
texfile.h:60:1: error: invalid use of template-name 'std::unordered_set' without an argument list
   60 | std::unordered_set const latexCharacters = {'#', '$', '%', '&', '\\', '^', '_', '{', '}', '~'};
      | ^~~
texfile.h:60:1: note: class template argument deduction is only available with '-std=c++17' or '-std=gnu++17'
In file included from /opt/local/include/gcc14/c++/unordered_set:41,
                 from symbolmaps.h:4:
/opt/local/include/gcc14/c++/bits/unordered_set.h:104:11: note: 'template<class _Value, class _Hash, class _Pred, class _Alloc> class std::unordered_set' declared here
  104 |     class unordered_set
      |           ^~~~~~~~~~~~~
texfile.h: In function 'void camp::texpreamble(T&, mem::list<std::basic_string<char, std::char_traits<char>, gc_allocator_ignore_off_page<char> > >&, bool, bool)':
texfile.h:74:59: error: 'latexCharacters' was not declared in this scope; did you mean 'escapeCharacters'?
   74 |   out << "\\def\\ASYprefix{" << escapeCharacters(outPath, latexCharacters) << "}" << newl
      |                                                           ^~~~~~~~~~~~~~~
      |                                                           escapeCharacters
make: *** [drawlabel.o] Error 1
In file included from acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/serializer.h:14,
                 from acextlibs/vectorgraphics/LspCpp/include/LibLsp/lsp/lsPosition.h:3,
                 from symbolmaps.h:11,
                 from errormsg.h:15,
                 from picture.cc:9:
acextlibs/vectorgraphics/LspCpp/include/LibLsp/JsonRpc/optionalVersion.h:5:14: error: 'template<class T> class boost::optional' conflicts with a previous declaration
    5 | using boost::optional;
      |              ^~~~~~~~
In file included from common.h:35,
                 from errormsg.h:13:
backports/optional/include/optional.hpp:600:7: note: previous declaration 'template<class T> class nonstd::optional_lite::optional'
  600 | class optional;
      |       ^~~~~~~~
In file included from drawelement.h:18,
                 from picture.h:15,
                 from picture.cc:10:
texfile.h:60:1: error: invalid use of template-name 'std::unordered_set' without an argument list
   60 | std::unordered_set const latexCharacters = {'#', '$', '%', '&', '\\', '^', '_', '{', '}', '~'};
      | ^~~
texfile.h:60:1: note: class template argument deduction is only available with '-std=c++17' or '-std=gnu++17'
In file included from /opt/local/include/gcc14/c++/unordered_set:41,
                 from symbolmaps.h:4:
/opt/local/include/gcc14/c++/bits/unordered_set.h:104:11: note: 'template<class _Value, class _Hash, class _Pred, class _Alloc> class std::unordered_set' declared here
  104 |     class unordered_set
      |           ^~~~~~~~~~~~~
texfile.h: In function 'void camp::texpreamble(T&, mem::list<std::basic_string<char, std::char_traits<char>, gc_allocator_ignore_off_page<char> > >&, bool, bool)':
texfile.h:74:59: error: 'latexCharacters' was not declared in this scope; did you mean 'escapeCharacters'?
   74 |   out << "\\def\\ASYprefix{" << escapeCharacters(outPath, latexCharacters) << "}" << newl
      |                                                           ^~~~~~~~~~~~~~~
      |                                                           escapeCharacters
make: *** [picture.o] Error 1
```

--- a/GUI/buildtool.py
+++ b/GUI/buildtool.py
@@ -7,8 +7,9 @@ from typing import Optional
 
 import click
 from PyQt5.uic import compileUiDir
+import os
 
-BUILD_ROOT_DIRECTORY = pathlib.Path(__file__).parent
+BUILD_ROOT_DIRECTORY = pathlib.Path(os.path.dirname(os.path.abspath(__file__)))
 sys.path.append(str(BUILD_ROOT_DIRECTORY.parent))
 
 import determine_pkg_info

--- a/Makefile.in
+++ b/Makefile.in
@@ -173,8 +173,8 @@ LSP_ROOT=@LSP_ROOT@
 
 $(LSP_BUILD_ROOT)/Makefile:
 	$(CMAKE) -B$(LSP_BUILD_ROOT) -S$(LSP_ROOT) \
-		-DCMAKE_CXX_FLAGS="-fPIE @OPTIONS@ @LSP_CXX_BUILD_FLAGS@ -I@ROOT_DIR_RELATIVE_TO_LSP@/$(GC)" \
-		@LSP_CMAKE_OPTIONS@
+		-DCMAKE_CXX_FLAGS="-fPIE @OPTIONS@ @LSP_CXX_BUILD_FLAGS@ \
+		-I@ROOT_DIR_RELATIVE_TO_LSP@/$(GC)" @LSP_CMAKE_OPTIONS@
 
 $(LSP_BUILD_ROOT)/liblspcpp.a: $(LSP_BUILD_ROOT)/Makefile
 	$(MAKE) -C $(LSP_BUILD_ROOT)
@@ -268,7 +268,7 @@ icons_rc.py: GUI/res/icons.qrc
 	-$(PYRCC) GUI/res/icons.qrc -o GUI/icons_rc.py
 
 GUI/xasyicons: GUI/res/icons.qrc GUI/buildtool.py
-	cd GUI && $(PYTHON) buildtool.py buildicons
+	-cd GUI && $(PYTHON) buildtool.py buildicons
 
 GUI/xasyqtui: $(UIFILES) GUI/buildtool.py
 	cd GUI && $(PYTHON) buildtool.py buildui

--- a/Makefile.in
+++ b/Makefile.in
@@ -236,7 +236,7 @@ $(RUNTIME_FILES:=.cc): %.cc: runtime.pl opsymbols.h runtimebase.in %.in
 
 GEN_PREOROCESSED_DEPFILE_BASE_ARGS = \
     $(PYTHON) gen_preprocessed_depfile.py \
-        --cxx-compiler=$(CXX) \
+        --cxx-compiler="$(CXX)" \
         --additional-raw-arguments="$(OPTS)" \
         --cxx-standard=$(subst gnu,,$(subst c++,,$(CXX_STANDARD)))
 

--- a/determine_pkg_info.py
+++ b/determine_pkg_info.py
@@ -48,7 +48,7 @@ def process_configure_ac_file(f):
     return result
 
 
-def determine_asy_pkg_info(ac_file):
+def determine_asy_pkg_info(ac_file="configure.ac"):
     """
     Returns a dictionary containing version information, if found
     from the ac_file specified

--- a/determine_pkg_info.py
+++ b/determine_pkg_info.py
@@ -9,17 +9,17 @@ import pathlib
 from typing import TextIO
 import re
 
-REGEXES: dict[str, re.Pattern[str]] = {
+REGEXES = {
     "base": re.compile(
         r"AC_INIT\(\s*\[\s*(.+)\s*]\s*,"  # pkg name
         + r"\s*\[\s*(.+)\s*]\s*,"  # pkg version
         + r"\s*\[\s*(.+)\s*]\s*\)"  # issue url
     ),
-    "asygl": re.compile(r"ASYGLVERSION\s*=\s*(.+)"),
+    "asygl": re.compile(r"ASYGLVERSION=(.+)"),
 }
 
 
-def process_configure_ac_file(f: TextIO) -> dict:
+def process_configure_ac_file(f):
     """
     Returns a dictionary containing version information, if found
     from the configure.ac file object
@@ -48,7 +48,7 @@ def process_configure_ac_file(f: TextIO) -> dict:
     return result
 
 
-def determine_asy_pkg_info(ac_file: os.PathLike = "configure.ac") -> dict:
+def determine_asy_pkg_info(ac_file):
     """
     Returns a dictionary containing version information, if found
     from the ac_file specified

